src/homework/client/src/components/List.js

```diff
@@ -0,0 +1,21 @@
+import ListItem from "./ListItem";
+import PropTypes from "prop-types";
+
+function List(props) {
+	return (
```

Comment:
prefer to use double space instead of a tab, it's really hard to read the code that way